src/tlo/methods/tb.py

```diff
@@ -20,7 +20,7 @@
 from tlo.util import random_date
 
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
+logger.setLevel(logging.CRITICAL)
```

Comment:
same here, we can configure this in cervical cancer analyses. This should be as it was
```suggestion
logger.setLevel(logging.INFO)
```